clab/config/transport.go

```diff
@@ -0,0 +1,73 @@
+package config
```

Comment:
wdyt @kellerza if we move the transport.go, ssh.go and sshkind.go to a `transport` package under the config dir? that should clean things up, but it will require some rework, since there will be some cycle dependencies at this point

Reply:
Can look into that, it should be fairly self-contained (except for ConfigSnippet, which might now have to become some sort of interface iso a struct)